fastlane/semver.sh

```diff
@@ -0,0 +1,368 @@
+#!/bin/sh
```

Comment:
this script is an unchanged copy from the ios repo, which was pulled in from the source noted below - it is just for checking that the version is increasing per semver standards